packages/modular-scripts/src/__tests__/esmView.test.ts

```diff
@@ -762,18 +816,17 @@ describe('modular working with an esm-view with custom index.html', () => {
     tempModularRepo = createModularTestContext();
     tempPackagesPath = path.join(tempModularRepo, 'packages');
     tempDistPath = path.join(tempModularRepo, 'dist');
-    runModularForTests(
-      tempModularRepo,
-      'add sample-esm-view-with-index --unstable-type esm-view --unstable-name sample-esm-view-with-index',
-    );
+    setupMocks(tempModularRepo);
 
-    await fs.copyFile(
-      path.join(__dirname, 'TestEsmView.test-tsx'),
-      path.join(tempPackagesPath, targetedView, 'src', 'index.tsx'),
-    );
-
-    await fs.mkdir(path.join(tempPackagesPath, targetedView, 'public'));
+    await addPackageForTests(targetedView, 'esm-view');
 
+    await Promise.all([
```

Comment:
Small change that barely makes a difference here but we should probably start doing more often: actually let async tasks run asynchronously when they don't affect each-other